### ISSUE TITLE
"Hide" servers

### ISSUE THREAD

**Issue by MrScopes** [Minehut/Meta#300]
# Feature request

**Checklist**
- I have read and agree to the README (found on https://github.com/Minehut/Meta)
- This is not a bug report, support request or plugin request
- I have searched the Issue Tracker to make sure this is not a duplicate issue
- This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
There should be a way to hide a server

**How the feature is useful**
This is a useful replacement for deleting servers. If users have unused servers they don't want to see on the panel page, they can just hide them and reveal at a later date if they want.
![image](https://user-images.githubusercontent.com/29683770/95496198-75a91580-096e-11eb-9110-d71c1aff7479.png)

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by LukasmanMHdude** [comment]
I feel like this is kinda a unneed feature. Just put it below the rest of your servers

**Comment by MiginsMC** [comment]
Yeah I dont really know anyone who uses > 5 servers and has issues with this. I dont really see it as a big problem but I guess it sounds cool

**Comment by ahoulton** [comment]
If you have more then a noticeable amount of servers, then drag your most used server to the top and leave the others, just saying as this an alternate way of doing this, even if Minehut developers did this, it would be one of the last things developed (maybe).

**Comment by drtshock** [comment]
The problem seems to be that you can have more servers on your list than needed. Maybe a better solution would be to allow users to delete servers, and then show how many in the bank you have to be able to create without paying for more server slots?

**Comment by MrScopes** [comment]
> The problem seems to be that you can have more servers on your list than needed. Maybe a better solution would be to allow users to delete servers, and then show how many in the bank you have to be able to create without paying for more server slots?

I would love server deletion. I've been suggesting this for several years now.
